index.html

```diff
@@ -2608,8 +2608,26 @@ <h4>Error Responses</h4>
         </p>
       </section>
     </section>
-  </section>
 
+    <!-- Discovery -->
+    <section>
+      <h2 id="discovery">Discovery</h2>
+      <p class="rfc2119-assertion" id="core-profile-discovery-1">
+        In order to a conform with the Core Profile, a Web Thing's Thing
+        Description [[wot-thing-description]] MUST be retrievable from a URL
```

Comment:
With the new terminology, what you want to say here is the Thing Description for a Thing must be available from a TD Server, given a URL provided with (at least) Direct introduction (which just means the user needs to provide the URL, somehow).

See my other comments.  Given we are talking greenfield devices, it might be best after all if we require .well-known and self-description, as long as we can resolve the privacy/security issues.